### ISSUE TITLE
[Adaptive Card-Schema Explorer]: The Content present under the adaptive card gets truncated in normal settings mode.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7888]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)
Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/%5D(https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))

Repro Steps:
1.Open above URL with V- credential.
2.Navigate to 'Schema Explorer' link using tab key and press enter key to activate it.
3.Navigate to "TextBlock" link present under CardElements from the left navigation using the tab key and activate it using Enter key.
4. Navigate to this content "wrap: false Lorem ipsum dolor sit amet, consectetur...." present under the adaptive card using Tab key.
5. Observe the issue.

Actual Result:
The content present after the "consectetur...." got truncated.

Expected Result:
The content after the "consectetur...." text will not get truncated in the normal settings mode. and user should be able to see the content clearly present on the page.

Observation:
Same issue existed on "Containers-->ColumnSet" path 
PFA: User is not able to see the info after contained.....mp4
Same issue existed on "input-->InputText" path.
PFA: riequest content is truncating in normal mode.png

User Impact:
All type of keyboard users will be affected, and they will get tensed if they cannot see the whole content present on the page.

MAs Reference:
https://liquid.microsoft.com/Web/Object/Read/ms.accessibility/Requirements/01.04.10

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

<img width="965" alt="truncated content in text block" src="https://user-images.githubusercontent.com/104482612/190582706-3195b02a-25b7-4f22-a7e0-366fc9198ccb.png">

https://user-images.githubusercontent.com/104482612/190581050-3ed91192-2cf0-43d3-8921-0649859d92ed.mp4

<img width="960" alt="riequest content is truncating in normal mode" src="https://user-images.githubusercontent.com/104482612/190581076-91628f54-db8f-4970-839d-9f3b12da93c0.png">

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by anna-dingler** [comment]
@Susmitha474 I am closing this bug as I believe it is a bug on Narrator.

Our text elements have a `wrap` property. For the examples you pointed out, `wrap` is `false`; therefore, we assigned the CSS property `text-overflow: ellipsis` to intentionally clip the text.

The scenario you described also occurs on the HTML examples on the mozilla site: https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow#examples

Let me know if you have any questions!

**Comment by vagpt** [comment]
As per the above comment, closing this issue.
